spec/datetimeformat.html

```diff
@@ -452,6 +556,26 @@
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-Intl.DateTimeFormat.prototype.formatToParts">
+      <h1>get Intl.DateTimeFormat.prototype.formatToParts</h1>
+
+      <p>
+        Intl.DateTimeFormat.prototype.formatToParts is an accessor property whose set accessor function is *undefined*. Its get accessor function performs the following steps:
+      </p>
+
+      <emu-alg>
+        1. Let _dtf_ be *this* value.
+        1. If Type(_dtf_) is not Object, throw a *TypeError* exception.
+        1. If _dtf_ does not have a [[boundFormatToParts]] internal slot, throw a *TypeError* exception.
```

Comment:
... not have a [[initializedDateTimeFormat]] internal slot.

this is because NumberFormat instances will have the same internal slot.